content/refguide/oracle.md

```diff
@@ -16,3 +16,7 @@ This problem is fixed in Oracle 11.2.0.4\. See [https://support.oracle.com/epmos
 Selecting DISTINCT attributes of type String and size > 2000 characters is not supported by Mendix due to a known Oracle limitation of selecting DISTINCT columns with a CLOB data type. When you run into this limitation, you may encounter an exception in the logs with a message like this:
 
 `Error Msg = ORA-06502: PL/SQL: numeric or value error: character string buffer too small`
+
+### Unlimited String Behavior
```

Comment:
Oracle also has this behavior for long-limited-strings. It starts at 2000 chars if I am right.

Reply:
true, i thought i added it but apparently not... now it is there